odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1525,12 +1525,40 @@ def _load_records_write(self, values):
             Note that we will only update unmodified fields. That will mimic the
             noupdate behavior on views having an ir.model.data.
         """
+        if not hasattr(self.pool, 'website_views_to_adapt'):
+            self.pool.website_views_to_adapt = set()
         if self.type == 'qweb':
             # Update also specific views
             for cow_view in self._get_specific_views():
                 authorized_vals = {}
                 for key in values:
-                    if cow_view[key] == self[key]:
+                    if key == 'inherit_id':
+                        # if inherit_id update, replicate change on cow view but
+                        # only if that cow view inherit_id wasn't manually changed
+                        #
+                        # A    A'   A"   B    B'               A    A'   B     B'
+                        # |    |    |                =>                 / \    |
+                        # D    D'   D"                                 D   D"  D'
+                        if self.inherit_id.id != values['inherit_id'] and \
+                           cow_view.inherit_id.key == self.inherit_id.key:
+                            if self.pool._init:
+                                self.pool.website_views_to_adapt.add((
+                                    # TODO: remove comment here and above
+                                    # A  -> self.inherit_id.id
+                                    # D  -> self.id
+                                    # A' -> cow_view.inherit_id.id
+                                    # D' -> cow_view.id
+                                    # B  -> values['inherit_id']
+                                    # B' -> ?? need to find it by knowing it is same key as B and same website_id as D'
+                                    cow_view.id,
+                                    values['inherit_id'],
+                                ))
+                            else:
+                                authorized_vals['inherit_id'] = self.search([
+                                    ('key', '=', self.browse(values['inherit_id']).key),
+                                    ('website_id', 'in', (False, cow_view.website_id.id)),
```

Comment:
No. It's still wrong. Don't refer to the `website_id` column in the `base` module.

Reply:
Done, I refactored the flow to exctract all website part in website module